src/engine/renderer/tr_local.h

```diff
@@ -2695,9 +2695,7 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 		growList_t lightmaps;
 		growList_t deluxemaps;
 
-		image_t    *fatLightmap;
-		int        fatLightmapSize;
-		int        fatLightmapStep;
+		image_t *internalLightmaps[ MAX_LIGHTMAPS ];
```

Comment:
It may be possible to use a dynamically allocated list and even drop the max but for now this is not worst than a large allocated image where smaller ones are concatenated. It's already smaller in memory and it is good enough.

Reply:
I can add this as a comment, though.

Reply:
Well, since it's only a list of pointers… I may even drop the list entirely, and then all globals for this.